src/pretix/settings.py

```diff
@@ -71,17 +71,12 @@
 db_backend = config.get('database', 'backend', fallback='sqlite3')
 if db_backend == 'postgresql_psycopg2':
     db_backend = 'postgresql'
-DATABASE_IS_GALERA = config.getboolean('database', 'galera', fallback=False)
-if DATABASE_IS_GALERA and 'mysql' in db_backend:
-    db_options = {
-        'init_command': 'SET SESSION wsrep_sync_wait = 1;'
-    }
-else:
-    db_options = {}
+if db_backend == 'mysql':
+    print("MySQL/MariaDB is not supported")
+    sys.exit(1)
```

Comment:
Instead of terminating application with `sys.exit`, please raise Django's [`ImproperlyConfigured`](https://docs.djangoproject.com/en/5.0/ref/exceptions/#improperlyconfigured).

Also don't call `print`. Pass the message to `ImproperlyConfigured` instead. By raising exeption, we let library user choose what to do, exiting program or recover with alternative settings.

Reply:
I have made the suggested changes.